### DISCUSSION TITLE
support ssh with password

### DISCUSSION THREAD

**Discussion by koolay** [umputun/spot#198] [Q&A]
Is support password for ssh connection?

**Comment by umputun** [comment]
I guess the question was about support for password authentication. Spot does not support password authentication and is unlikely to offer this feature in the future.

**Reply by koolay** [reply]
Why not support the password? It is more simple.

**Reply by umputun** [reply]
two reasons:

1. it is extra functionality with a new flag, like `--ask-passwd` or something like this. It should be developed, tested, and maintained.
2. Generally, I don't see a good use case for this. The tool is for running automated deployments, and involving such a manual step feels odd.

Saying this, I may think of some semi-valid use cases, for example, testing a playbook against a new host. So, if someone wants to offer a PR limited to the interactive password support, I'll consider it.

**Reply by koolay** [reply]
It is more simple using password if run only once on host.
